README.md

```diff
@@ -124,6 +124,22 @@ $ heroku config:set GO_INSTALL_PACKAGE_SPEC=./...
 $ git push heroku master
 ```
 
+## Usage with other vendoring systems
+
+If your vendor system of choice is not listed here, create `vendor/vendor.json`
+with the following contents, adjusted as needed for your project's root path.
+
+```json
+{
+    "comment": "For other heroku options see: https://devcenter.heroku.com/articles/go-dependencies-via-govendor#build-configuration",
+    "rootPath": "github.com/yourOrg/yourRepo",
+    "heroku": {
+        "sync": false,
+        "install": [ "./..." ],
```

Comment:
Please drop the `install` and `goVersion` lines as they are not strictly required for a minimal `vendor.json`, although may be required for some apps, but are documented [elsewhere](https://github.com/heroku/heroku-buildpack-go#govendor-specifics).